Documentation/docs-mobile/building-apps/build-properties.md

```diff
@@ -1079,6 +1079,15 @@ Specifies the name of the Resource
 file to generate. The default template sets this to
 `Resource.designer.cs`.
 
+## AndroidResourceDesignerClassModifier
+
+Specifies the class modifier for the intermediate `Resource` class which is
+generated. Valid values are `public`, `internal` and `private`.
```

Comment:
`private` isn't actually valid, at least not in C#:

```csharp
private class App {
	public static void Main() {}
}
```

results in:

```
error CS1527: Elements defined in a namespace cannot be explicitly declared as private, protected, protected internal, or private protected.
```

*In C#*, the only valid top-level visibility values are `public` and `internal`.  `private` is only usable for *nested* types:


```csharp
internal class Outer {
    private class Inner {}
}
```

Reply:
Regarding my [previous localization question](https://github.com/dotnet/android/pull/9057/files#r1662920507), the CS1527 error appears to have a similar list.  It's handled by hard-coding the entire list (sigh):

https://github.com/dotnet/roslyn/blob/828a6cfb476258ed751e75b15830409bf1847618/src/Compilers/CSharp/Portable/CSharpResources.resx#L2621

This means other languages can replace `or` as appropriate, e.g. German: https://github.com/dotnet/roslyn/blob/828a6cfb476258ed751e75b15830409bf1847618/src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf#L8841

```xml
<target state="translated">Die in einem Namespace definierten Elemente dürfen nicht explizit als "private", "protected", "protected internal" oder "private protected" deklariert werden.</target>
```

Maybe a generic XA1038 isn't a good idea, l10n-wise?